### PR TITLE
datree 0.7.1 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

relates to datreeio/datree#115

### CODE REVIEW

**Review by dawidd6 — DISMISSED** [review]

File: Formula/datree.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Datree < Formula
+  desc "CLI tool to run policies against Kubernetes manifests YAML files or Helm charts"
+  homepage "https://www.datree.io/"
+  url "https://github.com/datreeio/datree/archive/0.7.1.tar.gz"
+  sha256 "5a17495e60e7748d236af16a56485138d2e10bd769b7779033d0eb37e6f90fb7"
+  license "Apache-2.0"
+  head "https://github.com/datreeio/datree.git", branch: "staging"
```
Comment (dimabru) [inline Formula/datree.rb:7]: please change branch to `main`

Comment (chenrui333) [reply Formula/datree.rb]: you got it.

**Comment by chenrui333** [comment]
@dimabru, you might want to update the default branch on datree side?
```
datree:
  * HEAD: Use `branch: "staging"` to specify the default branch
```

**Comment by chenrui333** [comment]
I am gonna temporarily switch back to `staging` branch (as it is the listed default branch).

**Comment by romanlab** [comment]
@chenrui333 You're absolutely right, our default branch is staging. 

But we use staging because we want all pull requests to go automatically to the staging branch, so we can verify the version on our staging environment before we promote it to production. 

All stable, production level releases are on our main branch and therefore Homebrew should also point to the main branch to make sure it always references a stable version

**Comment by SMillerDev** [comment]
> All stable, production level releases are on our main branch and therefore Homebrew should also point to the main branch to make sure it always references a stable version

That's not what the `head` block is for though. `head` allows users to do a build from a recent git checkout (to validate patches for example). For the stable version we just use the tagged downloads.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**